### ISSUE TITLE
model tracking

### ISSUE THREAD

**Issue by gowthamvenkatsairam** [google-research/tapas#92]
how to track model training,when fine tuned on custom data ,to view loss function,f1 score etc.. on tensorboard..

**Comment by gowthamvenkatsairam** [comment]
the loss curve on tensorboard is able to plot only the final loss after finetuning,how can the loss be logged continously on tensorboard,where should i make the changes..

**Comment by eisenjulian** [comment]
Hi @gowthamvenkatsairam at the moment only the few scalars created automatically by the estimator API while training are added, namely batch loss and speed metrics. We are working on a change so that the main dataset metrics are also reported to tensorboard, but this will happen when running run_task_main in "predict_and_evaluate" mode. We recommend running this in parallel to the training mode so that this predictions are made for all checkpoints as they are saved. If you are interested in specific metrics feel free to add them in `_eval_for_set` method as a log, and soon there will be a helper method there adding directly to tensorboard.
